src/daemon/rpmostreed-transaction-types.c

```diff
@@ -191,13 +177,15 @@ package_diff_transaction_execute (RpmostreedTransaction *transaction,
 
   sysroot = rpmostreed_transaction_get_sysroot (transaction);
   upgrader = rpmostree_sysroot_upgrader_new (sysroot,
-					     self->osname,
-					     upgrader_flags,
-					     cancellable,
-					     error);
+                                             self->osname,
+                                             upgrader_flags,
+                                             cancellable,
+                                             error);
   if (upgrader == NULL)
     goto out;
 
+  origin = rpmostree_sysroot_upgrader_get_origin (upgrader);
```

Comment:
> Or alternatively, we define that the upgrader creates a new origin, and we simply drop `rpmostree_sysroot_upgrader_set_origin()`? (Does anything call it in your new code?)

That's already the case, though right? (This patch drops `rpmostree_sysroot_upgrader_set_origin()`). That said, I think I do also prefer the other way around as well (forcing a `dup` and then a `set`). Will update it to use that.